pkg/ingress/config_types.go

```diff
@@ -88,6 +88,10 @@ func (t *TargetGroupTuple) validate() error {
 	if t.ServiceName != nil && t.ServicePort == nil {
 		return errors.New("missing servicePort")
 	}
+
+	if (t.Weight != nil) && (*(t.Weight) < 0 || *(t.Weight) > 999) {
```

Comment:
Counsider using `aws.Int64Value()`

Reply:
review accepted, changes done. thanks for this 🙌